portfolio/src/main/webapp/getDataFromData.js

```diff
@@ -0,0 +1,2 @@
+
+    updateDataInclude();
```

Comment:
I don't like the name of this file. In this case, I'd probably go against the feedback I gave earlier and probably add `<script>updateDataInclude()</script>` to the bottom of your pages instead of putting this in a separate file.

While JS in HTML is usually frowned upon. Adding a single line or two to initialize your JS is not unreasonable.